08_files_lab/01.py

```diff
@@ -16,4 +16,4 @@
         for line in b:
             o.write(line)

```

Comment:
Cool. Another idea - you can replace the if with a single: `print >>o, line.rstrip()`